Tests/CombineXTests/Publishers/CombineLatestSpec.swift

```diff
@@ -98,6 +98,8 @@ class CombineLatestSpec: QuickSpec {
                 })
                 pub.subscribe(sub)
                 
+                subject0.send("A")
+                subject1.send("A")
                 100.times {
                     [subject0, subject1].randomElement()!.send("\($0)")
```

Comment:
This test has a 1/2^89 chance of failing and [it just happened](https://github.com/cx-org/CombineX/runs/708355747?check_suite_focus=true). Fixed by giving them initial value.

Reply:
haha